docs/getting-started/04-expose-microservice.md

```diff
@@ -0,0 +1,188 @@
+
+    - Select `orders-service (port: 80)` from the drop-down list in the **Service** column to indicate the Service name for which you want to create the API Rule.
+
+4. In the **Access strategies** section, leave only the `GET` and `POST` methods marked and the `noop` handler selected. This way you will be able to send the orders to the Service and retrieve orders from it without any token.
+
+5. Select **Create** to confirm the changes.
+
+    The message will appear on the screen confirming the changes were saved.
+
+6. In the API Rule's details view that opens up automatically, check if the API Rule status is `OK`. See if you can access the Service by selecting the HTTPS link under **Host** and adding the `/orders` endpoint at the end of it.
+
+> **NOTE:** For the whole list of endpoints available in the service, see its [OpenAPI specification](./assets/orders-service-openapi.yaml).
+
+</details>
+</div>
+
+### Call and test the microservice
+
+Follow these steps:
+
+> **CAUTION:** If you have a Minikube cluster, you must first add the IP address of the exposed k8s Service to the `hosts` file on your machine:
+>
+>  ```bash
+>  echo "$(minikube ip) orders-service.kyma.local" | sudo tee -a /etc/hosts
+>  ```
+
+1. Retrieve the domain of the exposed microservice and save it to the environment variable:
+
+  ```bash
+  export SERVICE_DOMAIN=$(kubectl get virtualservices -l apirule.gateway.kyma-project.io/v1alpha1=orders-service.orders-service -n orders-service -o=jsonpath='{.items[*].spec.hosts[0]}')
+  ```
+
+2. Run this command in the terminal to call the service:
+
+  ```bash
+  curl -ik "https://$SERVICE_DOMAIN/orders"
+  ```
+
+  The system returns a similar response:
+
+  ```bash
+  content-length: 2
+  content-type: application/json;charset=UTF-8
+  date: Mon, 13 Jul 2020 13:05:33 GMT
+  server: istio-envoy
+  vary: Origin
+  x-envoy-upstream-service-time: 37
+
+  []
+  ```
+
+3. Send a `POST` request to the microservice with sample order details:
+
+  ```bash
+  curl -ikX POST "https://$SERVICE_DOMAIN/orders" \
+    -H 'Cache-Control: no-cache' -d \
+    '{
+      "orderCode": "762727210",
+      "consignmentCode": "76272725",
+      "consignmentStatus": "PICKUP_COMPLETE"
+    }'
+  ```
+
+4. Call the microservice again to check the storage:
+
+  ```bash
+  curl -ik "https://$SERVICE_DOMAIN/orders"
+  ```
+
+  You should receive a similar response:
+
+  ```bash
+  HTTP/2 200
+  content-length: 73
+  content-type: application/json;charset=UTF-8
+  date: Mon, 13 Jul 2020 13:05:51 GMT
+  server: istio-envoy
+  vary: Origin
+  x-envoy-upstream-service-time: 6
+
+  [{"orderCode":"762727210","consignmentCode":"76272725","consignmentStatus":"PICKUP_COMPLETE"}]
+  ```
+
+  You can see the microservice returns the previously sent order details.
+
+5. Remove the [Pod](https://kubernetes.io/docs/concepts/workloads/pods/) created by the `orders-service` Deployment. Run this command and wait for the system to delete the Pod and start a new one:
```

Comment:
```suggestion
5. Remove the [Pod](https://kubernetes.io/docs/concepts/workloads/pods/) created by the `orders-service` Deployment. Run this command, and wait for the system to delete the Pod and start a new one:
```
I added a comma because of the two adjacent `and`'s to separate them and add more clarity.